app/controllers/api/api_controller.rb

```diff
@@ -17,7 +17,8 @@ class ApiController < ApplicationController
       Subjects::Selector::MissingSubjectSet,
       Subjects::Selector::MissingSubjects,                 with: :not_found
     rescue_from ActiveRecord::RecordInvalid,               with: :invalid_record
-    rescue_from Api::LiveProjectChanges,                   with: :forbidden
+    rescue_from Api::LiveProjectChanges,
+      Api::ExportDisabled,                                 with: :forbidden
```

Comment:
I don't think you need to expose this error as an api response. This error should only be detected in sidekiq after scheduling a dump to run and will just force sidekiq to retry and backoff over time, they'll just sit in the retry queue until we turn them back on.